src/components/BigNumber.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+
+import { getUnitPrefix } from '@/helpers/numbers';
+
+const BigNumber = ({ children, unit }) => {
+  const { number, prefix } = getUnitPrefix(children);
+  return `${number} ${prefix}${unit}`;
+};
+
+BigNumber.propTypes = {
+  children: PropTypes.number,
```

Comment:
if children are always a number why not pass them in as 'number' prop instead? also – does this component even allow multiple children?